openstack/blockstorage/v2/volumes/requests.go

```diff
@@ -0,0 +1,418 @@
+package volumes
+
+import (
+	"fmt"
+
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// CreateOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateOptsBuilder interface {
+	ToVolumeCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts contains options for creating a Volume. This object is passed to
+// the volumes.Create function. For more information about these parameters,
+// see the Volume object.
+type CreateOpts struct {
+	// The availability zone [OPTIONAL]
+	Availability string
```

Comment:
Thoughts about changing this to  `AvailabilityZone`? I know it was `Availability` in v1, but maybe now is a good time to make the name more intuitive.